pkg/controller/node/nodecontroller_test.go

```diff
@@ -761,6 +761,8 @@ func TestPodStatusChange(t *testing.T) {
 	}
 
 }
+var fakeNodeHandler *testutil.FakeNodeHandler
```

Comment:
I can make this change. I would have removed that anyways, just wanted to check if you are ok with core logic.